File: src/main/java/com/ject/studytrip/mission/domain/repository/MissionCommandRepository.java
package com.ject.studytrip.mission.domain.repository;

public interface MissionCommandRepository {
    boolean existsByStampIdAndCompletedIsFalseAndDeletedAtIsNull(Long stampId);

    long deleteAllByDeletedAtIsNotNull();

    long deleteAllByDeletedStampOwner();

    long deleteAllByMemberId(Long memberId);
}

File: src/main/kotlin/com/ject/studytrip/mission/domain/repository/MissionCommandRepository.kt
package com.ject.studytrip.mission.domain.repository

interface MissionCommandRepository {
    fun existsByStampIdAndCompletedIsFalseAndDeletedAtIsNull(stampId: Long): Boolean

    fun deleteAllByDeletedAtIsNotNull(): Long

    fun deleteAllByDeletedStampOwner(): Long

    fun deleteAllByMemberId(memberId: Long): Long
}
